src/algebra/order/monoid_lemmas_zero_lt.lean

```diff
@@ -232,12 +232,10 @@ lemma pos_mul_strict_mono_iff_pos_mul_mono_rev : pos_mul_strict_mono α ↔ pos_
 lemma mul_pos_strict_mono_iff_mul_pos_mono_rev : mul_pos_strict_mono α ↔ mul_pos_mono_rev α :=
 ⟨@mul_pos_strict_mono.to_mul_pos_mono_rev _ _ _ _, @mul_pos_mono_rev.to_mul_pos_strict_mono _ _ _ _⟩
 
-lemma pos_mul_reflect_lt.to_pos_mul_mono {α : Type*} [mul_zero_class α] [linear_order α]
-  [pos_mul_reflect_lt α] : pos_mul_mono α :=
+lemma pos_mul_reflect_lt.to_pos_mul_mono [pos_mul_reflect_lt α] : pos_mul_mono α :=
```

Comment:
FYI, this would be reverting what @YaelDillies did in #16447 (link: https://github.com/leanprover-community/mathlib/commit/660bde43158a8f5f7bcb605bd710ac3e5c55e81d#diff-9a2cc9f885f3ca126ef76ae5cff3d94471fc72ec9485f2f28d91431b32a323b8R235).

Reply:
The diff of #16447 is misleading. Most lemmas in the section here were generalized from `mul_zero_class` to `has_mul` + `has_zero`, and for some reason I kept those instances using `mul_zero_class`. So it shows up the diff even though it's actually everything around that changed.

This is not actually reverting my changes and looks like a fine move.